src/pages/go/quickstart.mdx

```diff
@@ -29,44 +29,36 @@ GOBIN in PATH
 
 <Steps>
 
-
 ### Import the library in your code
 
-In the main entrypoint of your program, you need to both add package
+In the main entrypoint of your program, add the package:
 
-```go
+```go filename="main.go"
 import (
 	autometrics "github.com/autometrics-dev/autometrics-go/pkg/autometrics/prometheus"
 )
 ```
 
-And then in your main function initialize the metrics
+And then in your main function initialize the metrics:
 
-```go
-	// Everything in BuildInfo is optional.
-	// You can also use any string variable whose value is
-	// injected at build time by ldflags.
+```go filename="main.go"
 	autometrics.Init(
-		nil,
+		"web-server",
```

Comment:
When using _prometheus_ in the import, this argument should be `nil` (or a pointer to a non-default Prometheus registry, but that's not really a "quickstart" at that point I'd guess). It is when the imported implementation is `otel` that the first argument here is a string.

It seems a bit tricky, but hopefully the compile error people are getting when using the wrong signature will help.

Reply:
ahh I see - ok I'll update it